1558751907717	Sync.LogManager	DEBUG	Flushing file log
1558751907734	Sync.LogManager	DEBUG	Log cleanup threshold time: 1557887907734
1558751907825	Sync.LogManager	DEBUG	Done deleting files.
1558751908674	Sync.SyncScheduler	DEBUG	Network link status changed to "changed". Offline?: false
1558751910565	Sync.SyncScheduler	DEBUG	Network link status changed to "changed". Offline?: false
1558751917013	Sync.SyncScheduler	DEBUG	Network link status changed to "changed". Offline?: false
1558751939475	Sync.SyncScheduler	DEBUG	Next sync in 600000 ms. (why=schedule)
1558752223547	Sync.SyncScheduler	DEBUG	Network link status changed to "down". Offline?: false
1558752223548	Sync.SyncScheduler	DEBUG	Clearing sync triggers and the global score.
1558752223555	Sync.SyncScheduler	DEBUG	Next sync in 600000 ms. (why=schedule)
1558752224380	Sync.SyncScheduler	DEBUG	Network link status changed to "changed". Offline?: false
1558752224646	Sync.SyncScheduler	DEBUG	Network link status changed to "up". Offline?: false
1558752224646	Sync.SyncScheduler	DEBUG	Network link looks up. Syncing.
1558752224655	Sync.Service	DEBUG	User-Agent: Firefox/67.0 (Linux x86_64) FxSync/1.69.0.20190521202118.desktop
1558752224655	Sync.Service	INFO	Starting sync at 2019-05-24 20:43:44 in browser session p3tpBcsAkZXJ
1558752224655	Sync.SyncScheduler	DEBUG	Clearing sync triggers and the global score.
1558752224660	Sync.Status	INFO	Resetting Status.
1558752224660	Sync.Status	DEBUG	Status.service: error.sync.failed => success.status_ok
1558752224677	Sync.Resource	WARN	GET request to https://sync-548-us-west-2.sync.services.mozilla.com/1.5/84144729/info/collections failed: [Exception... "The lookup of the hostname failed"  nsresult: "0x804b001e (NS_ERROR_UNKNOWN_HOST)"  location: "<unknown>"  data: no] No traceback available
1558752224677	Sync.Status	DEBUG	Status.sync: success.sync => error.login.reason.network
1558752224677	Sync.Status	DEBUG	Status.service: success.status_ok => error.sync.failed
1558752224678	Sync.ErrorHandler	ERROR	Sync encountered an error: [Exception... "The lookup of the hostname failed"  nsresult: "0x804b001e (NS_ERROR_UNKNOWN_HOST)"  location: "<unknown>"  data: no] No traceback available
1558752224680	Sync.SyncScheduler	DEBUG	Sync error count has exceeded 3; enforcing backoff.
1558752224680	Sync.SyncScheduler	DEBUG	Starting client-initiated backoff. Next sync in 6387608 ms.
1558752224680	Sync.SyncScheduler	DEBUG	Next sync in 6387608 ms. (why=client-backoff-schedule)
1558752224684	Sync.Service	DEBUG	Exception calling anonymous function: [Exception... "The lookup of the hostname failed"  nsresult: "0x804b001e (NS_ERROR_UNKNOWN_HOST)"  location: "<unknown>"  data: no] No traceback available
